tests/core/contracts/test_implicit_contract.py

```diff
@@ -70,13 +70,13 @@ def test_implicitcontract_transact_default(web3, math_contract, get_transaction_
     assert is_integer(start_count)  # Verify correct type
     # When a function is called that defaults to transact
     blocknum, starting_txns = get_transaction_count("pending")
-    with pytest.warns(DeprecationWarning,
-                      match='deprecated in favor of classic contract syntax') as warnings:
+    with pytest.warns(
```

Comment:
These tests were looking for a particular number of warnings which can't be sustainable as our dependencies will eventually issue warnings (as `eth-abi` did which broke these tests).